core/session/db.go

```diff
@@ -57,3 +57,32 @@ func (s *DB) Sessions() (Sessions, error) {
 	tx := s.db.Find(&res)
 	return res, tx.Error
 }
+
+func (s *DB) ClosePendingSessionsInHistory(chargeMeterTotal float64) error {
+	var res Sessions
+	if tx := s.db.Find(&res, map[string]interface{}{"finished": "0001-01-01 00:00:00+00:00", "Loadpoint": s.name}); tx.Error != nil {
```

Comment:
Warum nicht einfach nur die eine letzte Session raus suchen?

Reply:
ThinkEV hatte gewünscht die gesamte Historie anzupassen, damit zumindest die Charged Energy in den Stats passt.